unit_testing/test_reports.py

```diff
@@ -8,31 +8,50 @@
 from spinalcordtoolbox.image import Image
 from spinalcordtoolbox.reports.slice import Sagittal
 
+@pytest.fixture(scope="module")
+def im_base(path_in='t2/t2.nii.gz'):
+    # Base anatomical image
+    yield Image(path_in)
 
-def labeled_data_test_params(path_in='sct_testing_data/t2/t2.nii.gz',
-                             path_seg='sct_testing_data/t2/labels.nii.gz'):
-    """Generate image/label pairs for various test cases of
-    test_sagittal_slice_get_center_spit."""
-    im_in = Image(path_in)            # Base anatomical image
-    im_seg_labeled = Image(path_seg)  # Base labeled segmentation
-    assert np.count_nonzero(im_seg_labeled.data) >= 2, "Labeled segmentation image has fewer than 2 labels"
+@pytest.fixture(scope="module")
+def im_seg_labeled(path_seg='t2/labels.nii.gz'):
```

Comment:
Ah, the error is actually pretty simple here. 

Right now, `"t2/t2.nii.gz"` is referring to `"unit_testing/t2/t2.nii.gz"` when it should be referring to `"{TEST_DATA_DIR}/t2/t2.nii.gz"`. Ditto for `"t2/labels.nii.gz"`.

I think it would be nice if we shared the test data directory path to each of our tests via a fixture in `conftest.py`.

```python
# In conftest.py
@pytest.fixture
def sct_test_path():
    """Make test data directory path available to all tests as a fixture
    without having to import the function each time."""
    return sct_test_path()
```